packages/contracts-bedrock/src/L1/interfaces/ISystemConfigInterop.sol

```diff
@@ -28,18 +28,18 @@ interface ISystemConfigInterop {
     function gasPayingToken() external view returns (address addr_, uint8 decimals_);
     function gasPayingTokenName() external view returns (string memory name_);
     function gasPayingTokenSymbol() external view returns (string memory symbol_);
-    function initialize(
-        address _owner,
-        uint32 _basefeeScalar,
-        uint32 _blobbasefeeScalar,
-        bytes32 _batcherHash,
-        uint64 _gasLimit,
-        address _unsafeBlockSigner,
-        IResourceMetering.ResourceConfig memory _config,
-        address _batchInbox,
-        ISystemConfig.Addresses memory _addresses
-    )
-        external;
+    // function initialize(
```

Comment:
Why is this commented out?

Reply:
That was the initializer used by `ISystemConfig.sol`. Given that the initializer for `ISystemConfigInterop.sol` has an extra argument, the function couldn't be overriden and so there were 2 initializers, making it impossible to reference it via the interface here https://github.com/ethereum-optimism/optimism/blob/bc7f0b04f8024e55c49c94c337bed559803732ee/packages/contracts-bedrock/scripts/DeployImplementations.s.sol#L1250
Also had to exclude `ISystemConfigInterop.sol` in `check-interfaces.sh`

Reply:
Can we just delete this commented out code? I see that there's another `initializer` defined below, which I think is the right approach.